roles/lib_utils/filter_plugins/openshift_aws_filters.py

```diff
@@ -10,6 +10,37 @@
 class FilterModule(object):
     ''' Custom ansible filters for use by openshift_aws role'''
 
+    @staticmethod
+    def subnet_count_list(size, subnets):
+        """This function will modify ec2 tag list to an asg dictionary."""
+        items = {}
+        j = 0
```

Comment:
my 0.02 $ - will be nice to rename `j` to something more meaningful.

Same applies to the other variables like a,k

Reply:
would you be good with renaming it to counter1, counter2?